core/DCache.hpp

```diff
@@ -61,8 +111,8 @@ namespace olympia
 
         sparta::DataInPort<uint32_t> in_l2cache_ack_{&unit_port_set_, "in_l2cache_ack", 1};
```

Comment:
Add comment this port is receiving an ack from next level memory. If there's any possibility of connecting DCache to L3 or BIU directly, then this method name becomes confusing; consider changing method name if possibility exist.